spec/game_spec.rb

```diff
@@ -0,0 +1,35 @@
+require './lib/game'
+
+describe Game do
+  let(:game) { Game.new([1, 2, 3, 4], 'Daniel', 'Ronan') }
+
+  describe '#initialize' do
+    it 'Checks if the given player names are correct' do
+      expect(game.players).to eq('O' => 'Ronan', 'X' => 'Daniel')
+    end
+
```

Comment:
- [ ] You have checked if the player names are correct :clap: . However, you also need to check if the player names given are incorrect

Reply:
done